core-services/prow/02_config/openshift-agent-team/fleeting/_prowconfig.yaml

```diff
@@ -31,17 +31,3 @@ tide:
     - needs-rebase
     repos:
     - openshift-agent-team/fleeting
-  - includedBranches:
-    - main
-    labels:
-    - approved
-    - lgtm
-    missingLabels:
-    - backports/unvalidated-commits
-    - do-not-merge/hold
-    - do-not-merge/invalid-owners-file
-    - do-not-merge/work-in-progress
-    - jira/invalid-bug
-    - needs-rebase
-    repos:
-    - openshift-agent-team/fleeting
```

Comment:
any idea why this is being touched?

Reply:
ah, this is removing a duplicate entry

Reply:
There were 2 files that contained duplicate entries:

1. core-services/prow/02_config/openshift-agent-team/fleeting/_prowconfig.yaml
2. core-services/prow/02_config/openshift/operator-framework-olm/_prowconfig.yaml

I verified they were dups and removed them.